.travis/run.sh

```diff
@@ -123,7 +123,6 @@ for BOARD in $BOARDS; do
 			# Not currently built so use .bit instead
 			#cp ../third_party/misoc/build/*.xsvf $COPY_DEST
 			cp ../third_party/misoc/build/*.bit $COPY_DEST
-			cp ../build/output.*.log $COPY_DEST
```

Comment:
Getting all the files gets round this but having the logfile always the same name is probably good practice